modules/lib/maintainers.nix

```diff
@@ -258,6 +258,14 @@
     github = "LucasWagler";
     githubId = 32136449;
   };
+  mateusauler = {
+    name = "Mateus Auler";
+    email = "mateus@auler.dev";
+    github = "mateusauler";
+    githubId = 24767687;
+    keys =
+      [{ fingerprint = "A09D C093 3C37 4BFC 2B5A  269F 80A5 D62F 6EB7 D9F0"; }];
+  };
```

Comment:
As mentioned in the [Contributing](https://nix-community.github.io/home-manager/index.xhtml#_add_yourself_as_a_module_maintainer) section of the home-manager manual:

> If you are present in the nixpkgs maintainer list then you can use that entry. If you are not then you can add yourself to modules/lib/maintainers.nix in the Home Manager project.

Reply:
Missed this. Thank you!